java/pom.xml

```diff
@@ -76,7 +76,7 @@
     <maven.version>3.9.6</maven.version>
 
     <mockito.version>5.10.0</mockito.version>
-    <orc-format.version>1.0.0</orc-format.version>
+    <orc-format.version>1.0.0-SNAPSHOT</orc-format.version>
```

Comment:
? AFAIK, we didn't change anything relevant until now, do we need `-SNAPSHOT`?

Reply:
Thank you for the review. The spec of geometry type haven't been merged into orc format yet, so I created a private package for developing here.  Maybe it's a little bit early for code reviewing, let's waiting for the spec finished first.